website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -9,6 +9,12 @@ description: >-
 
 Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.
 
+## 2023-02-17
+
+<!-- BEGIN: TFC:only name:health-assessments -->
```

Comment:
since the API changes are not user-facing, i think we can remove the changes to this file?